docs/guides/studio_oauth.rst

```diff
@@ -1,25 +1,21 @@
 Enabling OAuth for Studio login
 ===============================
 
-Migration guide for edx.org (and anyone else following master) in converting Studio login to use OAuth.
-
-This is a temporary document for Arch-BOM.
+This is a migration guide for converting Studio login to use OAuth, for use in the Lilac to Maple upgrade.
 
 Background
 ----------
 
 As of Lilac, the Studio by default shares a session cookie with the LMS.  This either forces Studio to be on a subdomain of the LMS or the LMS to set its session cookie on a wide domain, which exposes it to a potentially large number of subdomains.
 
-By setting up Studio to use LMS's OAuth2-based single-sign-on (SSO), the cookies (and domains) can be decoupled to improve both flexibility and security.
+Maple's configuration assumes that Studio will use LMS's OAuth2-based single-sign-on (SSO). This means that the cookies (and domains) can be decoupled to improve both flexibility and security. However, there are a few steps to take to finish this configuration (otherwise Studio logins will not work.)
 
 Migration
 ---------
 
-Most of the configuration is already in place, and Studio and LMS just need to be configured in each environment to enable the new flow. (Devstack and sandboxes will autoconfigure for OAuth.)
-
-Migration involves simultaneously enabling OAuth and separating the session cookies for LMS and Studio. This effectively causes a logout for Studio users, although they'll still be logged into LMS and aside from a brief disruption during the mixed-config interval they should not experience many problems.
+Studio and LMS need to be configured in each environment to enable the new flow, with the exception of devstack and sandboxes (which will autoconfigure for OAuth.) Migration involves enabling OAuth and separating the session cookies for LMS and Studio. The session cookie split will require Studio users to log in again.
 
-For each deployed environment (stage, production, etc.):
+Follow these steps for each deployed environment (stage, production, etc.):
 
 #. Register an SSO OAuth2 client in LMS:
```

Comment:
Do we want to mention the management command that is used in devstack for setting this up?

Reply:
I could point to the `create_dot_application` management command that [devstack uses](https://github.com/edx/devstack/blob/caa5eac3742b110d7596399d72d31a8ad468e7a9/provision-ida-user.sh#L19) but I don't know if it's worth it since that command also creates `studio_worker` (which is then unused).

Reply:
I didn't remember how flexible the command was. If it isn't helpful, no need to send people that way.